### DISCUSSION TITLE
流浪者可以设置无限空居力吗 / Can Wanderer set infinite Kuugoryoku Points?

### DISCUSSION THREAD

**Discussion by xxtg666** [Melledy/Grasscutter#1992] [Ideas]
如题
as the title suggests

**Comment by Saint-Theana** [comment]
我也想知道

**Comment by ABA2396** [comment]
你可以获得流浪者吗？更新完之后我在图鉴里可以看到，但是用命令获取的时候提示说"无效的物品ID"，是我漏更新了啥么

**Reply by xxtg666** [reply]
用actions里的dev服务端

**Reply by ABA2396** [reply]
> 用actions里的dev服务端

奇怪，我下的就是刚合进去后的那个，之后再试试

**Reply by xxtg666** [reply]
还有resources也要记得更新

**Reply by ABA2396** [reply]
> 还有resources也要记得更新

更新了，我看提交是两周前的那个

**Comment by Simplxss** [comment]
不行 空居力不是服务器管的 是客户端自己计算的

**Reply by Saint-Theana** [reply]
Bad Mihoyo.
不让sf爽，让挂爽。

**Reply by JinShichang** [reply]
并不，3.8版本的冒险家活动可以无限空居力，应该是有对应的buff可以加空居力

**Comment by estlio** [comment]
可以使用外置辅助获取无限居空的能力 服务端应该没办法

**Comment by Len-syc** [comment]
空居力不是服务端的，是客户端单独计算的
